docs/src/documentation/05-development/04-migration-guides/01-v16.mdx

```diff
@@ -12,13 +12,49 @@ With this guide, we aim to walk through all the breaking changes and how they ca
 
 This version addresses especially a change in the design tokens, and tailwind classes.
 
-## Token and tailwind classes
+## Breaking changes
+
+### Heading component
+
+A new heading token was added: `title0`. This token is similar to the previously existing `title1` token. Therefore, `title1` token was updated to have new font size and line height values.
+
+With that, the `Heading` component now accepts a new value for the `type` prop: `"title0"`.
+
+Because of that, all occurrences of `Heading` with `type="title1"` must be replaced with `type="title0"`, to avoid the visual change.
+
+**Before:**
+
+```jsx
+<Heading type="title1">Title 1</Heading>
+```
+
+**Now:**
+
+```jsx
+<Heading type="title0">Title 0</Heading>
+```
+
+### Tokens
 
 A number of new design tokens were introduced and others were removed. This means that some of the tailwind classes have been updated to reflect these changes.
 
-### Border Radius
+#### Heading
+
+Some deprecated Heading tokens were removed. They have been deprecated since the first Tailwind release and were now finally removed. The equivalent new tokens are:
+
+- `fontSizeHeading<TYPE>` -> `heading<TYPE>FontSize`
+- `lineHeightHeading<TYPE>` -> `heading<TYPE>LineHeight`
+- `fontWeightHeading<TYPE>` -> `heading<TYPE>FontWeight`
 
-Four border radius tokens were removed and six new ones were added.
+Where `<TYPE>` can be `Display`, `DisplaySubtitle`, `Title1`, `Title2`, `Title3`, `Title4`, or `Title5`.
```

Comment:
Also `Title0`, right?

Reply:
and also `Title6` (?)

Reply:
`Title6` is missing, yes. Regarding `Title0`… it did not exist before, that's why I did not mention it. And on the section we already have the mention to the new title1 => title 0 relation.

So I will add `Title6`, but do you think I should add another mention to `Title1` now being `Title0`?

Reply:
Ok, I see now you're only talking about equivalences to the old tokens. So we can omit `Title0` then.